File: python/Status.py
from enum import Enum

class Status(Enum):
    def __init__(self, label: str):
          self.__label = label
    
    def get_label(self):
        return self.__label
    
    def set_label(self, label: str):
        return self.__label
    
    ACTIVE = "Active"
    HYBERNATED = "Hybernated"
    STOPPED = "Stopped"
    TERMINATED = "Terminated"

File: java/Status.java
public enum Status{
    ACTIVE("Active"),
    HYBERNATED("Hybernated"),
    STOPPED("Stopped"),
    TERMINATED("Terminated");
    
    private final String label;
    
    private Status(String label) {
        this.label = label;
    }
    
    public String getLabel(){
        return label;
    }
    
    public void setLabel(String label){
        this.label = label;
    }
}